src/index.js

```diff
@@ -26,90 +26,107 @@ import {
     get_point_and_line_annotations,
     mark_deprecated,
     update_distance_from_line_to_each_point,
-} from '../build/annotation_operators';
+} from "../build/annotation_operators";
 import {
     add_style_to_document,
     prep_window_html,
     build_id_dialogs,
     build_edit_suggestion,
-    build_confidence_dialog 
-} from '../build/html_builder';
+    build_confidence_dialog,
+} from "../build/html_builder";
 
-import $ from 'jquery';
+import $ from "jquery";
 const jQuery = $;
-window.$ = window.jQuery = require('jquery');
 
-const { v4: uuidv4 } = require('uuid');
+// Electron workaround: https://github.com/electron/electron/issues/254
+// eslint-disable-next-line @typescript-eslint/no-require-imports, no-undef
+window.$ = window.jQuery = require("jquery");
+
+import { v4 as uuidv4 } from "uuid";
```

Comment:
Notable case where I didn't follow the ESLint rule. It doesn't like mixing `import` and `require`, and as you can see I've adjusted the `uuid` import accordingly. 

I went through the `git` history so see why I was assigning to `window.$` and `window.jQuery`, and it appears that it was an electron fix. Going to leave it for now (with an updated comment as I'll certainly forget again).